### DISCUSSION TITLE
Start button position

### DISCUSSION THREAD

**Discussion by Lynx-Express** [Open-Shell/Open-Shell-Menu#1252] [General]
I've used Open-Shell in a way that Open-Shell start button is located in leftmost position in taskbar and other Windows 11 taskbar icons are in the middle. With latest builds of Open-Shell I seem to be unable to separate Windows start button and Open-Shell start button; they are on top of each other regardless of taskbar alignment. Am I missing a setting here or is this intended behavior? If this is intended, would it be possible to add a setting to position Open-Shell start button separately of Windows start button as it used to be?

**Comment by ge0rdi** [comment]
The point of `Open-Shell` start button is (and always was) to **replace** Windows one.

The behavior you observed in older builds was caused by incompatibility with Windows 11 (`Open-Shell` assumed that start button is always in the screen corner).
Newer builds properly replace start button on Windows 11 regardless of taskbar alignment.

If you want to use Windows start menu, you can simply `Shift`+click on start button (or `Shift`+`Win`).

**Reply by CornelisJ** [reply]
I agree with Lynx-Express that having the Start button of the Classic start menu on the left, and the Start button of the Windows 11 start menu in the middle, was a great feature. Today I updated from v4.4.170 to v4.4.190, and was annoyed to see that I could not separate the two start menu buttons. It would be really nice to have an option for placing the Start button of the Classic menu in the lower left corner, instead of replacing the Windows 11 start button.

In the mean time I will stay on v4.4.170.

**Reply by gitfirewall** [reply]
You can install the 4.4.180 too. 

Anyway it would be great if you could choose to have one or two start buttons.

**Reply by bjust620** [reply]
Concur, I'm going to stay with the older alleged "incompatible" version.

**Reply by xenos-github** [reply]
I too wish to have the open shell button in the corner. I hate the way the W11 buttons shuffle around when something else is opened. I like things to stay where I put them. 
In any case, it doesn't "replace" the W11 button; my customised button and the original W11 button appear superimposed.

**Reply by PeteBeet** [reply]
Except that the Windows start button on W7 had an arguably different function than on W8+. Certainly the button's behavior is different.

There is nothing inherently "wrong" with a W7-style button on the left plus the "new" button in center or right.

**Comment by Lynx-Express** [comment]
Well, I actually liked previous versions 'incompatibility' as a great feature because it gave me access to both start menus without using keyboard. Okay, thanks.

**Comment by 7zxkv** [comment]
I have seen this too and I think there could be an option to add the button on the side. But the real problem is that none of the themes are graphically compatible with windows 11. It would be necessary to combine the old menu with the windows11 graphics to make it look integrated into the system. Alas I have not yet seen a theme like that.

**Reply by bonzibudd** [reply]
I have a theme called [Fluent-Metro](https://github.com/bonzibudd/Fluent-Metro) that aims to do this for the menu (albeit with more Win10 style graphics). It's customizable for curved corners and iconography and such.

**Comment by TheRealPD** [comment]
Just thought to check to see what the nightly builds were like - & likewise this 'correction' is really annoying; as it was much better having it on the left...

...back to the old standard version I guess.

**Comment by srb24** [comment]
![start_menu_issue](https://github.com/Open-Shell/Open-Shell-Menu/assets/2411680/13ad8113-2101-4a13-b21a-4ba2eabe239a)
Hi, I'm on Win11 - windows key stopped working so I updated open shell, now my "classic" start menu has moved from the bottom left to over the top (none of the settings, seem to make a difference)  - is there a way to move it back or is there a previous version where the windows key works AND the button stays in the bottom left

**Reply by CornelisJ** [reply]
As indicated by user gitfirewall above, when you install Open-Shell v4.4.180 on Windows 11, the Classic start menu button will be in the lower left corner, while the Windows 11 start menu button will be in the middle. That is how I prefer it. As far as I know, when you want to go back from v4.4.190 to v4.4.180, you need to uninstall 4.4.190 before you can install 4.4.180.

I don't know about the Windows key that stopped working. I am on 4.4.180 now, and when I press the Windows key, the Windows 11 start menu is opened. I don't remember if this worked different in older versions of Open-Shell / Classic-Shell.

**Reply by srb24** [reply]
when I meant the win key "stopped working" I meant it opened the win11 start menu instead of the openshell as it did before (a win11 update broke it)- I will try and install earlier versions and report back....

**Reply by TheRealPD** [reply]
So, having opened up Open-Shell Menu Settings -> Clicking the "Show all settings" tickbox at the top --> & going to "Controls"...

...what have you got for "Windows Key opens:"?

Yeah, I'm sure there was at least a point when just the WK would be the Windows Menu, whilst Shift+WK was OpenShell... ...but have been using an XML file that has things how I like it for years.

**Comment by alan2675** [comment]
Does anyone prefer the menu button behavior of 4.4.190 over 4.4.180?
And if 4.4.190 behavior is "correct", then why is there a non-functional Settings option to place the icon in the corner? 
It seems like the "correct" behavior would be to let the users decide which behavior they want.  That would be the "Open" thing to do.
For now, I'm going back to 4.4.180 until a better option is offered.
Thanks

**Comment by f14tomcat2** [comment]
I much prefer the Open Shell Start Button be made independent, by option in settings, from the Windows Start Button.  The functionality of the 2 are distinct enough to be considered separate functions.  A term I'd like to use to describe why the Open Shell button should be on the left corner is "muscle memory".  The mind gets trained to move the mouse without looking and is expecting the button to be on the bottom left and the Win button to be in the middle area.  Aside from that, if "hover" is chosen as the "open" function, right clicking the combined center button to get to the "X" menu required a quick mouse movement off the button, else the Open Shell menu will open displacing the Win "X" menu.   Just too many combined functions when the buttons are combined.  Some may like the "Follow the start button in Win 11" but others, myself included, find in very cumbersome and annoying.  A simply setting to allow the "follow" or not would solve the whole issue.  Just one setting.  

Thanks.

**Comment by f14tomcat2** [comment]
I see that .191 is out.  Hopefully in .192 this request for the Start Button positioning can be addressed.   I remain on .180.

**Reply by PeteBeet** [reply]
This (and other responses) is absolutely correct. The way it worked in .170 was perfect for many scenarios. Windows 11 taskbar icons centered, classic Start Button on the left. Could not have been better.

Sorta ruined by the "update" and "whomever's" insistence that this is the way it "needs to be."

**Comment by f14tomcat2** [comment]
The Windows Start Menu is in essence a large Metro style display of apps, whereas Open Shell is a convenient common sense interface to user apps and system functions. As MS continues to take the Start Menu further and further off into left field, Open Shell remains stable and predictable. I know where I put things. To me, and I would imagine many others, Open Shell and the Windows Start Menu are 2 different worlds coexisting in one OS. Versions .180 and prior had a convenient separate of function. If it could be done there why can't it be optional for .190 and greater? Pic of .180. This is convenient and functional.


![](https://user-images.githubusercontent.com/142270598/262779176-0f6182e0-ee1f-428e-94bf-79ebc9a68d23.png)

**Reply by PeteBeet** [reply]
The best of both worlds!!! I go into taskbar settings on W11 and turn off the widgets (generally few folks care about these and some other so-called "features"). Cleans up the junk under/near the Open Shell start button.

Good luck convincing Microsoft that the Windows 11 UX is a step in the wrong direction. The more I use W10 with Open Shell, the more I dislike both the W11 and MacOS UX. Sorry folks, just my opinion. And of course, Open Shell is based on the W7 interface, which was pretty much the best Microsoft ever offered.

I will repeat as I've said before: I install Open Shell and have literally _never_ gotten a complaint that something was missing or harder to use. And we're talking about over 200 installations. Really.

**Reply by peterjay733** [reply]
My preferred Open Shell menu style is "Classic with two columns."

**Comment by Lynx-Express** [comment]
>  To me, and I would imagine many others, Open Shell and the Windows Start Menu are 2 different worlds coexisting in one OS. 

Exactly right.

**Reply by peterjay733** [reply]
Kinda. It's a shell, similar to what Nova7 (et. al.) does for Android, or the various shells available for desktop Linux. Definitely FEELS like a different world, I agree.

**Reply by PeteBeet** [reply]
Remember that Open Shell is open source. So perhaps some ambitious individual can modify the program, and add a option to move the start button "back." Just sayin'

Sorry to say I'm more of a hardware/network jockey so that's not in my job description.
